src/window/settings.rs

```diff
@@ -6,6 +6,26 @@ pub struct Settings {
     /// The initial size of the window.
     pub size: (u32, u32),
 
+    /// The initial position of the window.
+    ///
+    /// When the decorations of the window are enabled, Windows 10 will add some inivisble padding
+    /// to the window. This padding gets included in the position. So if you have decorations
+    /// enabled and want the window to be at 0,0 you would have to set the position to
+    /// -DPI_BORDER_X,-DPI_BORDER_Y.
+    ///
+    /// DPI_BORDER_X/DPI_BORDER_Y are the usual size of the padding, which changes based on the DPI of the display.
+    ///
+    /// On a 1920x1080 monitor you would have to set the position to -8,-2.
+    ///
+    /// For info on how you could implement positioning that supports all DPI monitors look at the
+    /// following WINAPI calls:
+    ///
+    /// * GetDpiForMonitor (with MDT_RAW_DPI)
+    /// * GetSystemMetricsForDpi (with SM_CXFRAME and SM_CYFRAME)
+    ///     
+    /// Note: this gets ignored on the web
```

Comment:
At least on my machine it sadly doesn't work like that. Probably because we are setting the "outer position" which includes the padding.

Reply:
I'm just pointing out that DPI should not be involved because the coordinates here represent a logical position (i.e. a position that is independent of the DPI or scale factor of the screen).

If that is not the case, then I'd consider it a `winit` bug.